src/app/components/search-results/search-results.component.html

```diff
@@ -1,6 +1,7 @@
 <section class="results js-results">
+  <div class="error__content" *ngIf="errorText">{{ errorText | translate }}</div>
   <div class="results__content" *ngIf="dataRecieved">
-    <div class="results__top">
+    <div class="results__top" (ngInit)="onResultsInit()">
```

Comment:
ah, is this how the init directive is used? does it override the normal init?

Reply:
It lets me get a callback when this part of the table gets rendered (so when dataRecieved (sic) becomes true and this div gets rendered... ah, how about I fix that typo). I use this to scroll past the blank space around the search box like we discussed.